editor/plugins/tiles/tiles_editor_plugin.cpp

```diff
@@ -562,7 +563,7 @@ TileSetEditorPlugin::TileSetEditorPlugin() {
 	editor->set_custom_minimum_size(Size2(0, 200) * EDSCALE);
 	editor->hide();
 
-	button = EditorNode::get_bottom_panel()->add_item(TTR("TileSet"), editor);
+	button = EditorNode::get_bottom_panel()->add_item(TTR("TileSet"), editor, ED_SHORTCUT_AND_COMMAND("bottom_panels/toggle_tile_set_bottom_panel", TTR("Toggle TileSet Bottom Panel"), KeyModifierMask::ALT | Key::S));
```

Comment:
TileMap and TileSet shortcuts should be close together I think.

Reply:
I've moved TileSet to <kbd>Alt + L</kbd> and Replication to <kbd>Alt + C</kbd>.